tests/testthat/test-shinytest2-tm_data_table.R

```diff
@@ -0,0 +1,39 @@
+test_that("e2e: tm_front_page initializes without errors", {
```

Comment:
@kartikeyakirar `tm_front_page` or `tm_data_table` ?
In the names of all tests : )

Reply:
opps , thanks.

Reply:
[e7c4228](https://github.com/insightsengineering/teal.modules.general/pull/719/commits/e7c4228e4aacf16d49f3a2031f46493c0cee31b6)